lib/server/routes/coinpayments.js

```diff
@@ -68,35 +103,35 @@ CoinpaymentsRouter.prototype.handleIPN = function (req, res) {
                 return res.sendStatus(200)
               });
           } else {
-            foundCredit.user = proc.user;
+            foundCredit.user = data.proc.user;
             foundCredit.paid = false;
             foundCredit.paid_amount = 0;
-            foundCredit.invoiced_amount = req.body.amount;
+            foundCredit.invoiced_amount = USD;
 
             return foundCredit.save()
               .then(() => res.status(204).end());
           }
-          })
-          .catch((err) => {
-            log.error('err status < 100: ', err);
-            return res.status(500).send(err)
-          })
+        })
+        .catch((err) => {
+          log.error('err status < 100: ', err);
+          return res.status(500).send(err)
+        })
       }
 
-      if (req.body.status === 100) {
+      if (req.body.status === '100') {
         return Credit.findOne({
           'data.txn_id': req.body.txn_id,
-          'user': proc.user
+          'user': data.proc.user
         })
         .then((credit) => {
           if (!credit) {
             const credit = new Credit({
-              paid_amount: req.body.amount,
-              invoiced_amount: req.body.amount,
-              user: proc.user,
+              paid_amount: USD,
+              invoiced_amount: USD,
+              user: data.proc.user,
               paid: true,
               payment_processor: constants.PAYMENT_PROCESSORS.COINPAYMENTS,
-              type: constants.CREDIT_TYPES.AUTO,
+              type: constants.CREDIT_TYPES.MANUAL,
```

Comment:
why `MANUAL`?

Reply:
Auto are ones that are subscription based, right? This isn't a subscription payment. They have to manually send the payments.